### ISSUE TITLE
Tracking not being added automatically

### ISSUE THREAD

**Issue by reboot-digital** [jameztrue/woocommerce-canadapost-ordercup#1]
I updated to the latest 1.4.2 version of the Canada Post plugin, also downloaded this plugin and activated it but after order cup set my order to complete and posted the tracking data as an order note the tracking data was not automatically added to the Tracking section.

Here was the order note which was added by ordercup at the time when order was set to complete:

Now ordercup posts 2 order notes, one private order note that contains the cost of the shipment at the end like this:

Carrier: Canada Post, Service: Expedited Parcel (1+ days), Tracking #: 8094975614265792 , Cost: 12.69

And a customer order note like this:

Carrier: Canada Post, Service: Expedited Parcel (1+ days), Tracking #: 8094975614265792

Hope that helps to track this glitch.

**Comment by jameztrue** [comment]
Thanks!  I've updated this plugin to match a little better with however the format of the note might be.  Hopefully OrderCup is actually adding the note before changing the status to 'completed'.  If it is adding the note afterward, this plugin will need to hook into an action like woocommerce_new_customer_note.  Anyways, please test this one out.  I am hoping this will be successful!

**Comment by reboot-digital** [comment]
I updated the plugin and tried again and still no go, perhaps they are adding the note afterward like you said?

**Comment by reboot-digital** [comment]
Hi, sorry to bother you.  Any chance you can attempt to hook into woocommerce_new_customer_note and see if that does the trick?  I would like to get this working and then I will post about it on the OrderCup customer support area as a great solution for woocommerce/ordercup/canadapost users and tell them about your plugin on CodeCanyon.  Thanks.

**Comment by jameztrue** [comment]
Yes, I will try to get to that soon, I am travelling this week. But the solution should work well. Will post it asap.
